packages/ui/readme.md

```diff
@@ -0,0 +1,28 @@
+# Penumbra UI
+
+This package contains the UI components for the Penumbra web.
+
+## Set up
+
+First, install the library:
+
+```bash
+npm i @penumbra-zone/ui
+```
+
+Then, configure the Tailwind in your project. Edit `tailwind.config.js`:
+
+```js
+export default {
+  content: [
+    // Parses the classes of the UI components
+    './node_modules/@penumbra-zone/ui/**/*.js',
+  ],
+};
```

Comment:
Info on how to consume the UI package externally. More on "why" here: https://github.com/prax-wallet/web/pull/45#discussion_r1647399875